src/Umbraco.Web/PublishedCache/ISystemDefaultCultureProvider.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Umbraco.Web.PublishedCache
+{
+    /// <summary>
+    /// Provides the system default culture.
+    /// </summary>
+    public interface ISystemDefaultCultureProvider
```

Comment:
@zpqrtbnk This isn't really a "System" thing, this is purely to do with 1:1 language variants

Reply:
will be renamed as part of U4-11227